src/unix/assets/86Box.spec

```diff
@@ -0,0 +1,123 @@
+# Fedora RPM spec file for 86Box including roms
+#
+# To create RPM files from this spec file, run the following commands:
+#  sudo dnf install rpm-build
+#  mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+#
+# copy this 86Box.spec file to ~/rpmbuild/SPECS and run the following commands:
+#  cd ~/rpmbuild
+#  sudo dnf builddep SPECS/86Box.spec
+#  rpmbuild --undefine=_disable_source_fetch -ba SPECS/86Box.spec
+#
+# After a successful build, you can install the RPMs as follows:
+#  sudo dnf install RPMS/$(uname -m)/86Box-3* RPMS/noarch/86Box-roms*
+
+%global romver 20220319
+
+Name:		86Box
+Version:	3.3
+Release:	1%{?dist}
+Summary:	Classic PC emulator
+License:	GPLv2
+URL:		https://86box.net
+
+Source0:	https://github.com/86Box/86Box/archive/refs/tags/v%%{version}.tar.gz
+Source1:	https://github.com/86Box/roms/archive/refs/tags/%{romver}.tar.gz
+
+BuildRequires: cmake
+BuildRequires: desktop-file-utils
+BuildRequires: extra-cmake-modules
+BuildRequires: gcc-c++
+BuildRequires: libFAudio-devel
+BuildRequires: libappstream-glib
+BuildRequires: libevdev-devel
+BuildRequires: libXi-devel
+BuildRequires: ninja-build
+BuildRequires: qt5-linguist
+BuildRequires: qt5-qtconfiguration-devel
+BuildRequires: qt5-qtbase-private-devel
+BuildRequires: qt5-qtbase-static
```

Comment:
Is there any particular reason why you pull static Qt5? Linux builds are not statically linked by default.

Reply:
The reason for this was because of this:
```
/builddir/build/BUILD/86Box-3.3/redhat-linux-build/src/qt/ui_autogen/EWIEGA46WW/../../../../../src/qt/qt_openglrenderer.hpp:31:14: fatal error: QtOpenGLExtensions/QOpenGLExtensions: No such file or directory
   31 | #    include <QtOpenGLExtensions/QOpenGLExtensions>
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
And when I did
```
sudo dnf provides '*QtOpenGLExtensions'
Last metadata expiration check: 0:18:15 ago on Mon 21 Mar 2022 09:32:58 CET.
mingw32-qt5-qtbase-5.15.2-5.fc35.noarch : Qt5 for Windows - QtBase component
Repo        : fedora
Matched from:
Other       : *QtOpenGLExtensions

mingw32-qt5-qtbase-5.15.2-6.fc35.noarch : Qt5 for Windows - QtBase component
Repo        : updates
Matched from:
Other       : *QtOpenGLExtensions

mingw64-qt5-qtbase-5.15.2-5.fc35.noarch : Qt5 for Windows - QtBase component
Repo        : fedora
Matched from:
Other       : *QtOpenGLExtensions

mingw64-qt5-qtbase-5.15.2-6.fc35.noarch : Qt5 for Windows - QtBase component
Repo        : updates
Matched from:
Other       : *QtOpenGLExtensions

qt5-qtbase-static-5.15.2-22.fc35.i686 : Static library files for qt5-qtbase
Repo        : fedora
Matched from:
Other       : *QtOpenGLExtensions

qt5-qtbase-static-5.15.2-22.fc35.x86_64 : Static library files for qt5-qtbase
Repo        : fedora
Matched from:
Other       : *QtOpenGLExtensions

qt5-qtbase-static-5.15.2-30.fc35.i686 : Static library files for qt5-qtbase
Repo        : updates
Matched from:
Other       : *QtOpenGLExtensions

qt5-qtbase-static-5.15.2-30.fc35.x86_64 : Static library files for qt5-qtbase
Repo        : @System
Matched from:
Other       : *QtOpenGLExtensions

qt5-qtbase-static-5.15.2-30.fc35.x86_64 : Static library files for qt5-qtbase
Repo        : updates
Matched from:
Other       : *QtOpenGLExtensions
```

It seems the only options I have are to use `qt5-qtbase-static` or `mingw{64,32}-qt5-qtbase`